src/Backend/Modules/MediaLibrary/Ajax/MediaItemFindAll.php

```diff
@@ -23,20 +24,31 @@ public function execute(): void
     {
         parent::execute();
 
-        /** @var MediaFolder|null $mediaFolder */
         $mediaFolder = $this->getMediaFolderBasedOnMediaGroup();
+        $aspectRatio = $this->getAspectRatio();
 
         // Output success message with variables
         $this->output(
             self::OK,
             [
-                'media' => $this->loadMediaItems($mediaFolder),
+                'media' => $this->loadMediaItems($mediaFolder, $aspectRatio),
                 'folder' => $mediaFolder !== null ? $mediaFolder->getId() : null,
                 'tab' => $this->selectedTab,
             ]
         );
     }
 
+    private function getAspectRatio(): ?AspectRatio
+    {
+        $aspectRatio = $this->get('request')->request->get('aspect_ratio');
```

Comment:
In think there are methods on request to get your variable casted (as a float in this case if I'm following the logic). This would allow you to avoid the `=== 'false'` check.